instances/atlas/brainAtlas/AAL1.jsonld

```diff
@@ -29,9 +29,6 @@
       {
         "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/AAL1_frontalLobe"
       },
-      {
```

Comment:
In general, this brainAtlas looks good, but could you explain why the instances are removed (it is not straight away clear to me.

Reply:
Thank you for the explanation. Then everything is in order.